weapons/defaultweaponclass.yaml

```diff
@@ -129,7 +129,7 @@ BlueFireballLauncher:
 		Shadow: true
 		Inaccuracy: 0c181
 		Image: missile01
-		HorizontalRateOfTurn: 8
+		VerticalRateOfTurn: 12
```

Comment:
Is replacing `HorizontalRateOfTurn` with `VerticalRateOfTurn` intended? Sounds plausible to me but it's better to ask.